serial_test/Cargo.toml

```diff
@@ -11,4 +11,5 @@ categories = ["development-tools::testing"]
 
 [dependencies]
 lazy_static = "1.2"
-parking_lot = "0.9"
\ No newline at end of file
+parking_lot = "0.9"
+serial_test_derive = "~0.2.0"
```

Comment:
Can you please re-add the `path` bit as well? https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies indicates (reading between the lines a little) that this should make us use the in-tree `serial_test_derive` when building from this repo, but then be able to use a version for pushing to crates.io. Thanks!